locale/en/corteza-webapp-compose/block.yaml

```diff
@@ -290,6 +290,8 @@ recordList:
       label: 'Chart drill down'
   tooltip:
     deleteSelected: Delete selected records
+    moduleFields: Selecting more than 10 columns and/or fields of type user or record will impact performance
+    recordPerPage: Displaying more than 15 record per page will impact performance
```

Comment:
Lets avoid specific numbers, so for example `Displaying a large number of records per page will impact performance`